docs/site/Interceptor-generator.md

```diff
@@ -0,0 +1,160 @@
+---
+lang: en
+title: 'Interceptor generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Interceptor-generator.html
+---
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+Adds a new [interceptor](Interceptors.md#global-interceptors) class to a
+LoopBack application.
+
+```sh
+lb4 interceptor [--global] [--group <group>] [<name>]
+```
+
+### Arguments and options
+
+`<name>` - Name of the interceptor to create as an argument to the command. If
+provided, the tool will use that as the default when it prompts for the name.
+
+`--global` - Optional flag to indicate a global interceptor (default to `true`).
+Use `--no-global` to set it to `false`.
```

Comment:
> --no-global 

Is this the standard in CLI options..to place  'no-' in front of an option like `global`?

Perhaps `not-global` or `non-global` ?

Reply:
It's a built-in convention for yeoman.